app/layout.tsx

```
@@ -1,35 +1,49 @@
+          content="The only Lorem Ipsum alternative you need"
+        />
         <meta property="og:type" content="website" />
         <meta property="og:image" content="/opengraph.png" />
         <meta name="twitter:card" content="summary_large_image" />
         <meta name="twitter:image" content="/opengraph.png" />
-        <meta property="twitter:title" content="Slopsum | on god fr"></meta>
-        <meta property="twitter:description" content="The only Lorem Ipsum alternative you need"></meta>
+        <meta
+          property="twitter:title"
+          content="Slopsum | on god fr"
+        ></meta>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)